### PR TITLE
readme

### PR DESCRIPTION
**Pull request by gidden**
Hey everyone,

I've added a barebones readme. Please take at least a cursory look, especially on the `Contributing` and `Conflicts` sections. I know the conflict stuff sounds silly now, but I've found that having a agreed-upon resolution mechanism in place is never a bad thing.

A simple +1  from all of you is fine with me for pulling this in.

### CODE REVIEW

**Comment by coroa** [comment]
+1

@sjpfenninger @ojdo : you might want to watch toolchest! I just caught @gidden 's pullrequest by chance.

**Comment by coroa** [comment]
i see the split into develop/master as one which makes sense, but only at a somewhat later stage, once the first users are in place. Since you're pulling into master i guess, you somewhat agree. Let's not forget to implement this split later!

**Comment by sjpfenninger** [comment]
I'm not a fan of `develop` branches. Why not pull into master and tag releases off of master? That's kind of the default I expect when I reach a project's repo. We can always add a `release` or `stable` branch if necessary.

**Comment by gidden** [comment]
Hey @sjpfenninger, the rationale for having `develop` branches is explained [here](http://nvie.com/posts/a-successful-git-branching-model/)

In short, `develop` is used as a back-stop in case some issue is found with a code before release. That is, `develop` can be considered unstable, but `master` is stable.

For example, after this and the License PR, I would make a `develop` branch and make it the default. While I've found this development model quite useful, I don't want it to hinder us at the beginning. Perhaps @ojdo can weigh in and I'll update the PR accordingly.

Finally, note that there is also a License PR!

File: README.rst [hunk 1]
```diff
@@ -103,3 +95,51 @@ On Windows
...
+; see https://github.com/paetzke/py-autopep8.el
+(require 'py-autopep8)
+(add-hook 'python-mode-hook 'py-autopep8-enable-on-save)
+```
+
+Conflicts between Contributors
+------------------------------
+
+Should any conflicts arise in the course of the project, without any other
+particularly good solutoin, resolution will follow a majority rule on the
```
Comment (ojdo) [inline README.rst]: typo: solut**io**n

**Comment by ojdo** [comment]
@coroa thanks for the mention, repo is on watch now.
@gidden right now I would opt for the simpler "no `develop` branch" model, and simply creating releases whenever we have the feeling of having reached a new milestone. I wager that the PR peer reviews will do a good enough job to keep `master` reasonably stable. Just my 2ct, though :smile:
The explicit conflict resolution vote seems fine to me (apart from the typo 'solutoin').

**Comment by gidden** [comment]
ok, I believe I've addressed everyone's comments, this should be good to go

**Comment by coroa** [comment]
My suggestion would be to start with a simple master only work-flow,
all the while the pr's are used as gatekeepers, and use tagging to
indicate major feature set changes as @sjpfenninger had already
suggested.

When we later notice any instabilities creeping in, we can easily switch
to the more conservative separate develop/master approach.

**Comment by coroa** [comment]
ah, I should read ALL my mails before replying :)

i don't have write access, so can't merge!

**Comment by gidden** [comment]
You've been added. There's a team called "toolchest-pullers" on the
openmod-initiative group. Everyone in it has Admin access (thus should be
able to add others). You have to be a "member" of the github org to be in
the team I think. Once we get some additions to the repo, I think we should
advertise it more broadly on the list serv.

**Pull request merged**